modules/perception/obstacle/fusion/probabilistic_fusion/probabilistic_fusion_test.cc

```diff
@@ -28,9 +28,10 @@ namespace perception {
 TEST(ProbabilisticFusionTest, probabilistic_fusion_test) {
   FLAGS_work_root = "modules/perception";
   FLAGS_config_manager_path = "./conf/config_manager.config";
-  AINFO << "start probabilistic_fusion_test\n";
+  AERROR << "start probabilistic_fusion_test\n";
```

Comment:
Not error?

Reply:
good point. Accidentally added the debug log. reverted in the new CL. thanks.